src/honey_badger.rs

```diff
@@ -60,6 +60,9 @@ pub struct HoneyBadger<Tx, NodeUid> {
     decrypted_selections: BTreeMap<NodeUid, Vec<u8>>,
     /// Ciphertexts output by Common Subset in an epoch.
     ciphertexts: BTreeMap<u64, BTreeMap<NodeUid, Ciphertext<Bls12>>>,
+    /// Those shares in `received_shares` that could not be verified on reception due to the absense
```

Comment:
Is keeping track about those even necessary? When we receive the ciphertext, doesn't that mean that none of the existing shares have been verified yet? I.e. won't this set always either be equal to the senders whose shares we've already received (if we don't have the ciphertext yet), or empty (if we have it)?

(Also typo: `absence`)

Reply:
This is indeed the case. Thanks for spotting this.